<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="dogsList"/>
   </item>
   <item>
    <widget class="QPushButton" name="showAdoptionListButton">
     <property name="text">
      <string>Show Adoption List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="adoptButton">
     <property name="text">
      <string>Adopt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextDogButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
